BeforeDayBegins.md

```diff
@@ -31,18 +31,18 @@ soon enough
 
 a day, or two,   
 a night, or three,  
-it will be  
+but not now
 
 a day, a night  
-will pass until  
-dusk is here  
+they too shall pass 
+dusk will come 
 
 We'll light up the ovens to bake our bread  
 and look, where there's ashes and smoke  
 blossom will heal up the wounds of war   
 life will grow   
 
-Someday we'll welcome a newborn child  
-and look, they will laugh back at us  
-for clinging to stories of long-gone times,  
-war-torn past  
+Someday our children will shake their heads  
```

Comment:
Ale z tą wersją to jak miałem problem tak mam, bo nawet znając tekst to mi chwilę zajmuje zrozumieć czemu one _will shake their heads_
stawiając się zaś w sytuacji osoby która tekstu nie zna to w ogóle trochę widzę to:
![image](https://user-images.githubusercontent.com/4687592/212419472-79c81798-cfd3-4f00-83bf-fa1c86dea417.png)
bo gdyż ponieważ _shake their heads_ nie jest w angielskim tak oczywistą sugestią kręcenia głową na nie, jak to jest po polsku.

Reply:
jednocześnie nadal nie jestem zadowolony z mojej i Łazownika alternatywy, ale lepszej jeszcze nie wykminiłem

Reply:
hmm, czy ja wiem, czy nie taką oczywistą? to dosyć popularny zwrot smh